src/main/resources/io/seqera/wave/spack/spack-builder-dockerfile.txt

```diff
@@ -16,32 +16,33 @@ ENV \
   AWS_ROLE_ARN=${AWS_ROLE_ARN} \
   AWS_WEB_IDENTITY_TOKEN_FILE=${AWS_WEB_IDENTITY_TOKEN_FILE}
 
-RUN mkdir -p /opt/spack-env \
-&&  sed -i -e 's;compilers:;compilers::;' \
-         -e 's;^ *flags: *{};    flags:\n      cflags: -O3\n      cxxflags: -O3\n      fflags: -O3;' \
-         /root/.spack/linux/compilers.yaml \
-&& cd /opt/spack-env \
-&& spack config add config:install_tree:/opt/software \
-&& spack config add concretizer:unify:true \
-&& spack config add concretizer:reuse:true \
-&& spack config add packages:all:target:[{{spack_arch}}] \
-&& printf "  view: /opt/view\n" >> /opt/spack-env/spack.yaml
+# Set up Spack environment
+RUN cd /opt/spack-env \
```

Comment:
The `/opt/spack-env` directory doesn't seem to exist in the current `spack/ubuntu-jammy:develop` image, why are we removing the `mkdir`?

```
[alberto@fedora]$ podman run --rm -it --entrypoint bash spack/ubuntu-jammy:develop 
root@b26a32e6d4e9:~# 
root@b26a32e6d4e9:~# cd /opt/spack-env
bash: cd: /opt/spack-env: No such file or directory
```

Reply:
The directory gets created at line 3 of the Dockerfile by this `COPY` command:
```
COPY spack.yaml /opt/spack-env/spack.yaml
```